### DISCUSSION TITLE
美其名曰，老板！，和旧时代的地主有何区别？

### DISCUSSION THREAD

**Discussion by lettermessenger** [WeNeedHome/SummaryOfLoanSuspension#514] [General]
不都是一样对劳动人民进行压迫、剥削。那么旧时代打地主，打的到底是什么？

**Comment by zxcs001** [comment]
老板？

别搞笑了，资本家们就一群狗，白手套们。你以为牵着狗的人是谁啊……

还在搞，有错的都是资本家们，青天大老爷们都是被蒙蔽的。你是活在清朝么？

扶清灭洋？砍了教民的狗头？

21世纪都快过一半了，你连启蒙运动还没到？

**Reply by lettermessenger** [reply]
你跑题了 我只是再说 老板 和地主有何区别 ，兄弟 别乱开枪

**Comment by zhaokai5520** [comment]
你以为资本家能翻天？青天大老爷呢？
